run.sh

```diff
@@ -87,7 +87,7 @@ print_configuration() {
 
 build_all() {
   export SODIUM_USE_PKG_CONFIG=1
-  cargo build $CARGO_PROFILE_ARG
+  cargo +nightly-2021-08-04 build $CARGO_PROFILE_ARG
```

Comment:
If https://github.com/tezedge/tezedge/pull/900 gets merged before this PR please remember to remove these references to nightly.